.github/workflows/repository_dispatch.yml

```diff
@@ -0,0 +1,33 @@
+name: Repository dispatch on main
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+env:
+  BIOME_WEBSITE_REPO: biomejs/website
+  BIOME_PUSH_ON_MAIN_EVENT_TYPE: biome-push-on-main-event
+
+jobs:
+  repository-dispatch:
+    name: Repository dispatch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch event on push
+        if: ${{ github.event_name == 'push' }}
+        uses: peter-evans/repository-dispatch@v3
+        with:
+          token: ${{ secrets.BIOME_REPOSITORY_DISPATCH }}
```

Comment:
This is the PAT token we use and it should be stored as a repository secret and the name should match (I used the name `BIOME_REPOSITORY_DISPATCH`, we can changed the name but they should be kept the same).